src/Stratis.SmartContracts.Executor.Reflection/ReflectionVirtualMachine.cs

```diff
@@ -69,57 +112,83 @@ public class ReflectionVirtualMachine : ISmartContractVirtualMachine
 
                 return VmExecutionResult.Error(gasMeter.GasConsumed, result.Exception.InnerException ?? result.Exception);
             }
-            else
-                this.logger.LogTrace("[CREATE_CONTRACT_INSTANTIATION_SUCCEEDED]");
+
+            this.logger.LogTrace("[CREATE_CONTRACT_INSTANTIATION_SUCCEEDED]");
+            
+            this.logger.LogTrace("(-):{0}={1}", nameof(contractAddress), contractAddress);
```

Comment:
This log line needs to be combined with the one below. So remove this one and include the parameters in the bottom one like so:

```
this.logger.LogTrace("(-):{0}={1}, {2}={3}", nameof(contractAddress), contractAddress, nameof(gasMeter.GasConsumed), gasMeter.GasConsumed);
```